src/logging.cc

```diff
@@ -1444,7 +1702,96 @@ vector<string> LogCleaner::GetOverdueLogNames(
 
   return overdue_log_names;
 }
+#endif
+
+#if defined(GLOG_OS_WINDOWS) && defined(UNICODE)
+bool LogCleaner::IsLogFromCurrentProject(
```

Comment:
Most of the wide string function variants are just copy pasted versions of the original implementations. Maintaining such code will be particularly difficult.

Therefore, please avoid duplicating functionality. Existing code can be almost completely reused by using templates.

Reply:
I will try to modify it using templates and submit it here. We will check that modification together. You know, this is really complex.